File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Ocsa/OCS0103U00GrdOCS0115Info.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Ocsa
{
    [Serializable]
    public class OCS0103U00GrdOCS0115Info
    {
        private String _hangmogCode;
        private String _inputGwa;
        private String _inputPart;
        private String _gwaName;
        private String _jundalTableOut;
        private String _jundalPartOut;
        private String _movePart;
        private String _jundalTableInp;
        private String _jundalPartInp;
        private String _gwaNameOut;
        private String _gwaNameInp;
        private String _movePartName;
        private String _startDate;
        private String _endDate;
        private String _hospCode;
        private String _hangmogStartDate;
        private String _rowState;

        public String HangmogCode
        {
            get { return this._hangmogCode; }
            set { this._hangmogCode = value; }
        }

        public String InputGwa
        {
            get { return this._inputGwa; }
            set { this._inputGwa = value; }
        }

        public String InputPart
        {
            get { return this._inputPart; }
            set { this._inputPart = value; }
        }

        public String GwaName
        {
            get { return this._gwaName; }
            set { this._gwaName = value; }
        }

        public String JundalTableOut
        {
            get { return this._jundalTableOut; }
            set { this._jundalTableOut = value; }
        }

        public String JundalPartOut
        {
            get { return this._jundalPartOut; }
            set { this._jundalPartOut = value; }
        }

        public String MovePart
        {
            get { return this._movePart; }
            set { this._movePart = value; }
        }

        public String JundalTableInp
        {
            get { return this._jundalTableInp; }
            set { this._jundalTableInp = value; }
        }

        public String JundalPartInp
        {
            get { return this._jundalPartInp; }
            set { this._jundalPartInp = value; }
        }

        public String GwaNameOut
        {
            get { return this._gwaNameOut; }
            set { this._gwaNameOut = value; }
        }

        public String GwaNameInp
        {
            get { return this._gwaNameInp; }
            set { this._gwaNameInp = value; }
        }

        public String MovePartName
        {
            get { return this._movePartName; }
            set { this._movePartName = value; }
        }

        public String StartDate
        {
            get { return this._startDate; }
            set { this._startDate = value; }
        }

        public String EndDate
        {
            get { return this._endDate; }
            set { this._endDate = value; }
        }

        public String HospCode
        {
            get { return this._hospCode; }
            set { this._hospCode = value; }
        }

        public String HangmogStartDate
        {
            get { return this._hangmogStartDate; }
            set { this._hangmogStartDate = value; }
        }

        public String RowState
        {
            get { return this._rowState; }
            set { this._rowState = value; }
        }

        public OCS0103U00GrdOCS0115Info() { }

        public OCS0103U00GrdOCS0115Info(String hangmogCode, String inputGwa, String inputPart, String gwaName, String jundalTableOut, String jundalPartOut, String movePart, String jundalTableInp, String jundalPartInp, String gwaNameOut, String gwaNameInp, String movePartName, String startDate, String endDate, String hospCode, String hangmogStartDate, String rowState)
        {
            this._hangmogCode = hangmogCode;
            this._inputGwa = inputGwa;
            this._inputPart = inputPart;
            this._gwaName = gwaName;
            this._jundalTableOut = jundalTableOut;
            this._jundalPartOut = jundalPartOut;
            this._movePart = movePart;
            this._jundalTableInp = jundalTableInp;
            this._jundalPartInp = jundalPartInp;
            this._gwaNameOut = gwaNameOut;
            this._gwaNameInp = gwaNameInp;
            this._movePartName = movePartName;
            this._startDate = startDate;
            this._endDate = endDate;
            this._hospCode = hospCode;
            this._hangmogStartDate = hangmogStartDate;
            this._rowState = rowState;
        }

    }
}

File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/ocsa/OCS0103U00GrdOCS0115Info.java
package nta.med.data.model.ihis.ocsa;

import java.util.Date;

public class OCS0103U00GrdOCS0115Info {
	private String hangmogCode            ;                                         
	private String inputGwa               ;                                         
	private String inputPart              ;                                         
	private String gwaName                ;
	private String jundalTableOut        ;                                         
	private String jundalPartOut         ;                                         
	private String movePart               ;                                         
	private String jundalTableInp        ;                                         
	private String jundalPartInp         ;                                         
	private String gwaNameOut            ;
	private String gwaNameInp            ;
	private String movePartName          ;
	private Date startDate              ;                                          
	private Date endDate                ;                                          
	private String hospCode               ;
	private Date hangmogStartDate      ;
	public OCS0103U00GrdOCS0115Info(String hangmogCode, String inputGwa,
			String inputPart, String gwaName, String jundalTableOut,
			String jundalPartOut, String movePart, String jundalTableInp,
			String jundalPartInp, String gwaNameOut, String gwaNameInp,
			String movePartName, Date startDate, Date endDate, String hospCode,
			Date hangmogStartDate) {
		super();
		this.hangmogCode = hangmogCode;
		this.inputGwa = inputGwa;
		this.inputPart = inputPart;
		this.gwaName = gwaName;
		this.jundalTableOut = jundalTableOut;
		this.jundalPartOut = jundalPartOut;
		this.movePart = movePart;
		this.jundalTableInp = jundalTableInp;
		this.jundalPartInp = jundalPartInp;
		this.gwaNameOut = gwaNameOut;
		this.gwaNameInp = gwaNameInp;
		this.movePartName = movePartName;
		this.startDate = startDate;
		this.endDate = endDate;
		this.hospCode = hospCode;
		this.hangmogStartDate = hangmogStartDate;
	}
	public String getHangmogCode() {
		return hangmogCode;
	}
	public void setHangmogCode(String hangmogCode) {
		this.hangmogCode = hangmogCode;
	}
	public String getInputGwa() {
		return inputGwa;
	}
	public void setInputGwa(String inputGwa) {
		this.inputGwa = inputGwa;
	}
	public String getInputPart() {
		return inputPart;
	}
	public void setInputPart(String inputPart) {
		this.inputPart = inputPart;
	}
	public String getGwaName() {
		return gwaName;
	}
	public void setGwaName(String gwaName) {
		this.gwaName = gwaName;
	}
	public String getJundalTableOut() {
		return jundalTableOut;
	}
	public void setJundalTableOut(String jundalTableOut) {
		this.jundalTableOut = jundalTableOut;
	}
	public String getJundalPartOut() {
		return jundalPartOut;
	}
	public void setJundalPartOut(String jundalPartOut) {
		this.jundalPartOut = jundalPartOut;
	}
	public String getMovePart() {
		return movePart;
	}
	public void setMovePart(String movePart) {
		this.movePart = movePart;
	}
	public String getJundalTableInp() {
		return jundalTableInp;
	}
	public void setJundalTableInp(String jundalTableInp) {
		this.jundalTableInp = jundalTableInp;
	}
	public String getJundalPartInp() {
		return jundalPartInp;
	}
	public void setJundalPartInp(String jundalPartInp) {
		this.jundalPartInp = jundalPartInp;
	}
	public String getGwaNameOut() {
		return gwaNameOut;
	}
	public void setGwaNameOut(String gwaNameOut) {
		this.gwaNameOut = gwaNameOut;
	}
	public String getGwaNameInp() {
		return gwaNameInp;
	}
	public void setGwaNameInp(String gwaNameInp) {
		this.gwaNameInp = gwaNameInp;
	}
	public String getMovePartName() {
		return movePartName;
	}
	public void setMovePartName(String movePartName) {
		this.movePartName = movePartName;
	}
	public Date getStartDate() {
		return startDate;
	}
	public void setStartDate(Date startDate) {
		this.startDate = startDate;
	}
	public Date getEndDate() {
		return endDate;
	}
	public void setEndDate(Date endDate) {
		this.endDate = endDate;
	}
	public String getHospCode() {
		return hospCode;
	}
	public void setHospCode(String hospCode) {
		this.hospCode = hospCode;
	}
	public Date getHangmogStartDate() {
		return hangmogStartDate;
	}
	public void setHangmogStartDate(Date hangmogStartDate) {
		this.hangmogStartDate = hangmogStartDate;
	}
	
	

}
